### ISSUE TITLE
The accuracy fluctuation is relatively large.

### ISSUE THREAD

**Issue by zhaowwenzhong** [deepinsight/insightface#352]
I am training with arcface. The entire training set has at least 2 photos each, with 12w people and a total of 6 million photos. Lr: 0.1, WD: 0.0005. Save a model every time you train an epoch and test it. It is now found that the training accuracy rises steadily, but the test test accuracy fluctuates greatly. What is the reason for this, and what parameters are generally tried to adjust when encountering this phenomenon?
@nttstar  

lfw_112x112:86.38+2.03
lfw_112x112:50.00+0.00
lfw_112x112:50.00+0.00
lfw_112x112:96.68+0.85
lfw_112x112:98.75+0.55
lfw_112x112:50.00+0.00
lfw_112x112:98.93+0.48
lfw_112x112:94.72+0.95
lfw_112x112:98.80+0.53
lfw_112x112:97.95+0.50
lfw_112x112:97.58+0.85
lfw_112x112:50.00+0.00
lfw_112x112:50.00+0.00
lfw_112x112:50.00+0.00

**Comment by nttstar** [comment]
try decreasing learning rate
